### PR TITLE
Done - Slider and rating component

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,7 +45,6 @@ <h1 class="title">Build fast, responsive sites with WOW UI</h1>
         <!-- Hero Right Section Start -->
         <div class="hero-right">
             <div class="hero-img-cont">
```
Comment (aayushjain-code) [inline index.html:47]: try to use better naming conventions for classes

**Review by aayushjain-code — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (MuskanChhatrasal) [reply index.html:47]: Okayy!! Will work on it

File: components/Slider.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.slider {
+  -webkit-appearance: none;
+  width: 20%;
+  height: 25px;
```
Comment (05him) [inline components/Slider.css:4]: try using rems for heights and widths

File: Componentss/ratingSlider.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+    <section class="main-container main">
+        <div class="pd-1-lr pd-1-bot">
+
+            <!-- Slider -->
+
+            <h1 class="mg-point8-tb title sub-head hover-underline-animation">Slider</h1>
+            <p>An slider is a graphical representation of a user or the user's character or persona. Lorem ipsum dolor
+                sit, amet consectetur adipisicing elit. Laboriosam, repellendus?
+
```
Comment (05him) [inline Componentss/ratingSlider.html:121]: extra empty line  inside the < p > tag maybe you were trying to give a empty line  after the < p > tag

**Review by 05him — APPROVED** [review]
Loved the WoW momo based UI .. 
just few minute suggestion from my side which you can go through 
else its good to go

File: components/Slider.css (hunk 2 above)
Comment (MuskanChhatrasal) [reply components/Slider.css:4]: Okayy!! Will follow that.
Thankyou!!

File: Componentss/ratingSlider.html (hunk 3 above)
Comment (MuskanChhatrasal) [reply Componentss/ratingSlider.html:121]: It's by mistake. Will remove that.
Thankyou!!

**Pull request merged**